database.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Observations: The database connection is established using procedural MySQLi, which is not ideal for larger applications due to lack of scalability and maintainability.

Recommendations: Use PDO or MySQLi with OOP to manage database connections. Consider using a singleton pattern to manage database connections efficiently.